### DISCUSSION TITLE
有没有可能实现「全文翻译」时，悬停鼠标出现原文？

### DISCUSSION THREAD

**Discussion by crysishacker** [immersive-translate/immersive-translate#1817] [Ideas]
### 新功能 | New Feature

1.通常双语翻译页面都变得乱七八糟，很难看
2.只显示中文模式时
3.悬停鼠标出现原文

### 为什么您想要添加这些功能 | Why you want to add these features

1.
2.
3.

**Comment by keyboard3** [comment]
现在就是这样的呀 
![image](https://github.com/immersive-translate/immersive-translate/assets/13177744/6047dfbd-98af-4795-a598-23e4cd19ffa5)

**Reply by crysishacker** [reply]
你是指「鼠标悬停」那个选项吗？是按ctrl键才能切换吗？我指的是不按任何按键，直接显示
但是如果我选择了「直接翻译该段」，没办法悬停时自动还原英文

**Reply by crysishacker** [reply]
另外我指的是，已经全文翻译了（仅显示中文），但是有些地方翻译感觉不对，鼠标移上去，显示原文，鼠标移开，又变回中文。

**Reply by crysishacker** [reply]
又或者，我觉得更加好的是，鼠标移动上去，弹出一个框显示英文原文，移动开又消失

**Comment by keyboard3** [comment]
这类需求通常建议双语阅读，如果那个站点糟糕，你可以给让开发来适配这个站点。 

短期不会对仅译文模式下做更多的交互效果增强

**Reply by crysishacker** [reply]
那好吧，反正也不是多强的需求
